### ISSUE TITLE
How do i change the height of the bar, it is just a bit too thick for my monitor.

### ISSUE THREAD

**Issue by ItsTerm1n4l** [Seme4eg/dotfiles#1]
I cant find how to change the height, and the settings i would ordinarily adjust aren't affecting it so something must be taking up more space and overriding it.
Other than this issue, this is one of the best `eww` bars I have ever seen, thank you.

**Comment by Seme4eg** [comment]
Hi, thanks for the first issue on my dots repo, glad to know someone uses it)

TLDR: content of center and right blocks makes eww window grow, use 'slim' class on 'centerbox' widget to save up on paddings.

Eww's height is determined also by the height of its content. The window (whole eww status bar you are seeing) contains 3 blocks - container left, right and center. If you replace right and center containers with 'box' and set `:height "20px"` you will see that the eww window indeed became smaller in height. So in order to make the whole window not as tall you need to make several adjustments to font/icon sizes, paddings/margins etc and spend some time no debugging what exactly makes eww grow in height. I did try to make it not as tall without sacrificing the looks, but it started to harm my eyes when i was trying to actually see what's shown on my statusbar. 

If you are ready to sacrifice a bit on looks you can use 'slim' class (its documented where to add it in the code itself)

**Comment by ItsTerm1n4l** [comment]
Ok I will have a look at it, i had already tried the "slim" setting but the normal one just looks way cooler and I dont think the "slim" one is actually any thinner, it is just closer to the top of the screen.

**Comment by Seme4eg** [comment]
exactly, that's what i meant when i said 'use slim to save up on paddings', it doesn't change fonts sizes or any other sizes, just removes paddings and shadows.

**Comment by Seme4eg** [comment]
managed to squeeze like 6px more, changes in master branch

**Comment by ItsTerm1n4l** [comment]
Thank you, this is much better and doesn't look out of proportion on my monitor.
![20240305_08h19m27s_grim](https://github.com/Seme4eg/dotfiles/assets/118511626/d528d30a-213b-4c55-8793-7d410db34168)

**Comment by Seme4eg** [comment]
hmm, i think the icon on the left or your bar is making it (bar) much larger than it should be, also take a look at layout script of mine, that shows your current layout near the network widget. And you don't have battery level cuz you are on stationary pc? And your notifications text is misaligned, tho on my monitors it is fine. And for some reasons you don't have bluetooth widget and updates widget. Also in case you replaced 'X' and 'O' in workspace widget for '=' for the reason that those looked awful maybe you need to install the font i am using. Cuz in my case the bar looks like this. Also for some reason you don't have brightness widget.
![image](https://github.com/Seme4eg/dotfiles/assets/24554850/cd74ba23-6e6c-45ec-80e3-fd4864e7429a)

**Comment by Seme4eg** [comment]
for updates counts you need to check why your 'updates_count' command in eww/right.yuck isn't working. For backlight i have script in eww scripts folder, you can try to debug it as well. You're missing out on many info widgets. Also i forcepushed to master to rename the commits that relate to this issue. And 1 more to come to fix backlighting battery indication when charging.

**Comment by ItsTerm1n4l** [comment]
Im on a PC so I don't have a battery level and i dont use backlight programs, i removed some modules from the bar for a less cluttered look,  I'm just using `-` and `=` for a change, and it is using the same font as you.
